share/wake/lib/system/job.wake

```diff
@@ -44,6 +44,11 @@ tuple Runner =
   Fn:           Job => Result RunnerInput Error => Result RunnerOutput Error
 
 # Create new Runner given pre- and post-hooks around an existing Runner
+# The `score` argument is of type `Plan → Result Double String` and is called by `runJob` to produce a score representing the priority of a runner with respect to the given Plan. For example, if `plan.getPlanResources` returns list `("python/3.7.1", Nil)` then the RISC-V runner probably cannot provide that resource and should return something like `Fail RISCVRunner: cannot provide resource: python/3.7.1". If the plan resources is `("riscv-tools/2019.02.0", Nil)` then the `score` function should return `Pass 1.0` or some other positive number.
```

Comment:
As we talked offline, let's:

1. Wrap the lines so that they're not super long. My personal preference is 80 chars, but I feel that that's increasingly less popular. Let's at least keep it under 120, which seems to be the max that GitHub shows without scrolling
2. Trim some of the content here. The Quickref can keep more of the more detailed info. This can focus on more of the specifics about what this argument is about.
3. Can we format this in a way such that Sphinx is able to understand that these are arguments of the function? @terpstra, does the Sphinx plugin include Javadoc-like comment syntax like the `@param` in this example: https://en.wikipedia.org/wiki/Javadoc#Examples